django/contrib/admin/models.py

```diff
@@ -113,6 +113,9 @@ def get_change_message(self):
                         messages.append(gettext('Added.'))
 
                 elif 'changed' in sub_message:
+                    sub_message['changed']['fields'] = [
+                        gettext(field_name) for field_name in sub_message['changed']['fields']
+                    ]
                     sub_message['changed']['fields'] = get_text_list(
                         sub_message['changed']['fields'], gettext('and')
                     )
```

Comment:
Isn't that going to inhibit the previous lines you have added ? Setting sub_message['changed']['fields'] over again twice in a row. 

Thanks for sharing some of your insight.

Reply:
Nope. In the second case `sub_message['changed']['fields']` is fetched again. This could certainty be made less clunky though as `sub_message['changed']['fields']` is referenced 4 times in 6 lines...